File: 卡码/KamaCoder/Js/test01.js
const rl=require("readline").createInterface({input:process.stdin});
const iter=rl[Symbol.asyncIterator]();
const readline=async ()=>(await iter.next()).value;

async function main(){
  while(line=await readline()){
    const [a,b]=line.split(" ").map(Number);
    console.log(a+b);
  }
}

main();

File: 卡码/KamaCoder/Python/test01.py
#1. A+B问题I
import sys

def main():
	for line in sys.stdin:
		a,b=map(int,line.split())
		print(a+b)

def main1():
	while True:
		try:
			a,b=map(int,input().split())
			print(a+b)
		except:
			break

if __name__=="__main__":
	main1()